src/View/InstallQuickInput.ts

```diff
@@ -88,12 +88,31 @@ export async function showInstallQuickInput() {
     return (input: MultiStepInput) => pickVersion(input, state);
   }
 
+  async function requestPrerequisites(toolchainEnv: ToolchainEnv) {
+    const result = await toolchainEnv.prerequisitesAsync();
+    if (result === true) {
+      Balloon.info('Backend toolchain list has been successfully updated.');
+    } else {
+      Balloon.error('Failed to update Backend toolchain list.');
+    }
+  }
+
   async function pickVersion(input: MultiStepInput, state: Partial<State>) {
     if (state.toolchainEnv === undefined || state.toolchainType === undefined) {
       throw Error('toolchainenv  is undefined.');
     }
     // TODO(jyoung): Support page UI
-    const toolchains = state.toolchainEnv.listAvailable(state.toolchainType, 0, 10);
+    let toolchains: Toolchain[];
+    try {
+      toolchains = state.toolchainEnv.listAvailable(state.toolchainType, 0, 10);
+    } catch (err) {
+      const answer = await vscode.window.showWarningMessage(
+          'Prerequisites has not been set yet. Do you want to set it up now?', 'Yes', 'No');
+      if (answer === 'Yes') {
+        requestPrerequisites(state.toolchainEnv);
+      }
+      return;
```

Comment:
IMHO user would feel nice if it is shown that installing workflow has been canceled after pressing `No` button. It can be added after merging this pr, or remaining as todo :)

Reply:
Is this what you said? I'm not sure which one is more appropriate, warning or info. I will make another PR for this one. :)
![initial_loading_prerequisites_no_message](https://user-images.githubusercontent.com/7223627/173043444-a5956866-f516-4392-9804-ce1b9f38c284.gif)

Reply:
Exactly it is! Thank you :)